src/toolbar.scss

```diff
@@ -33,8 +32,7 @@ $title-toolbar-height: 2.75rem;
 }
 
 @mixin toolbarInfo() {
```

Comment:
Should we have mixin names in kebab-case or in the camelCase? cc @InnaAtanasova

Reply:
they should even be prefixed by `fd-` but it was done by other developer and we can ignore it for now. I had planned a big cleanup in fund-styles for the mixins and the naming. It's getting a bit messy and we should def agree on some best practices, document them and start enforcing them in PR reviews. But for now, since it's not introduced in this PR we can leave it as it is.